types/koa-ratelimit/index.d.ts

```diff
@@ -38,7 +37,7 @@ declare namespace KoaRatelimit {
         /**
          * The database powering the backing rate-limiter package.
          */
-        db: Redis | RedisClient | Map<any, any>;
+        db: Redis | Map<any, any>;
```

Comment:
It turns out that kop-ratelimit is just a wrapper on async-ratelimit: https://github.com/koajs/ratelimit/blob/master/limiter/redis.js

async-ratelimit is typed, and does not support `redis`, only `ioredis`.